posix/wait.c

```diff
@@ -24,8 +24,10 @@
 __pid_t
 __wait (__WAIT_STATUS_DEFN stat_loc)
 {
+  /*
   __set_errno (ENOSYS);
-  return -1;
+  return -1; */
+  return 200;
```

Comment:
This wait.c was initially created by me to test the wait signal when running the bash program with my previous fork() implementation. The return value was set to 200 for test purpose only, and was not intended for the final implementation. Therefore, this piece of code should be removed, or commented out for further implementation.

Reply:
Whoops missed this, will fix.